package.json

```diff
@@ -22,7 +22,7 @@
     "build": "webpack",
     "watch": "webpack --watch",
     "test": "npm-run-all --aggregate-output --parallel --continue-on-error test:* lint:*",
-    "test:mocha": "mocha --colors",
+    "test:mocha": "mocha --colors --require babel-polyfill --require babel-register",
```

Comment:
Argh, running into that was bound to happen as we have more tests.
We can't do exactly that because that's babeling everything including server tests, and that means we could easily ship something incompatible with Node 4 or 6 on the server...

... But! I've been working on solving this and I'm almost there: https://github.com/thelounge/lounge/pull/1726. There is one more dependency I rely on that needs releasing (the PR I opened to fix their repo is merged, they _just_ need to release) but after that I'll be all set. That means that our client tests will run with the same config than our font-end builds, while the server tests remain unaffected.
I'm pretty excited about it because it brings a few good things, and opens the door to full-feature browser testing. Also I know it is not 100% ideal yet, but we can always improve from there.

(Also if they don't release before your PR is ready, we can find a workaround to not block your PR on it!)

Reply:
Yeah, that's a valid reason. 

But ... Surely the easiest solution is just having 2 different run configurations for client and server tests?

Reply:
(this is now out of this pr and in #1864)

Reply:
> But ... Surely the easiest solution is just having 2 different run configurations for client and server tests?

That is pretty much what #1726 does, with test configuration running behind the (almost) same Webpack config than our client builds.